src/test/transaction_tests.cpp

```diff
@@ -240,27 +306,34 @@ BOOST_AUTO_TEST_CASE(tx_invalid)
 
             TxValidationState state;
             fValid = CheckTransaction(tx, state) && state.IsValid();
+            if (!fValid) {
```

Comment:
join these lines and don't use the local variable `fValid` (it means 'the test file was malformed' above and 'the transaction failed validation' here).